packages/tao/src/commands/ngcli-adapter.ts

```diff
@@ -981,8 +1089,9 @@ export function wrapAngularDevkitSchematic(
           r.eventPath === 'workspace.json'
         ) {
           saveWorkspaceConfigurationInWrappedSchematic(host, r);
+        } else {
+          host.write(r.eventPath, r.content);
         }
```

Comment:
We handle the host.write inside of saveWorkspaceConfiguration now, so having it here would be 2 writes and fail w/o `workspace.json` present.